src/com/core/tictactoe/HumanPlayer.java

```diff
@@ -0,0 +1,17 @@
+package com.core.tictactoe;
+
+public class HumanPlayer extends Player {
+
+    private CommandLineUI commandLineUI;
+
+    public HumanPlayer(String sign) {
+        super(sign);
+        this.commandLineUI = new CommandLineUI(System.out, System.in);
```

Comment:
Could you maybe construct the CommandLineUI before injecting it to the HumanPlayer?